WalletWasabi.Fluent/CrashReport/Views/CrashReportWindow.axaml

```diff
@@ -0,0 +1,45 @@
+<Window xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:vm="using:WalletWasabi.Fluent.CrashReport.ViewModels"
+        xmlns:controls="clr-namespace:WalletWasabi.Fluent.Controls"
+        x:Class="WalletWasabi.Fluent.CrashReport.Views.CrashReportWindow"
+        Width="600" Height="500"
+        MinWidth="600" MinHeight="500"
+        MaxWidth="600" MaxHeight="500"
+        Title="Wasabi Wallet - Crash Reporting"
+        WindowStartupLocation="CenterScreen"
+        ExtendClientAreaToDecorationsHint="True"
+        ExtendClientAreaChromeHints="PreferSystemChrome"
+        x:Name="CrashReporterWindow"
+        x:CompileBindings="True"
+        x:DataType="vm:CrashReportWindowViewModel"
+        CanResize="False">
```

Comment:
It is not possible to move the window because of this. I think you can remove this. Moving the window is way more important than an incorrect cursor icon on the edges.

Also maximized button should be hidden but Iv found Avalonia does not support it yet... https://github.com/AvaloniaUI/Avalonia/issues/3422